pkgpanda/http/config.py

```diff
@@ -0,0 +1,15 @@
+from pkgpanda import constants
+
+
+DCOS_ROOT = constants.install_root
+DCOS_CONFIG_DIR = constants.config_dir
+DCOS_REPO_DIR = constants.repository_base
+DCOS_ROOTED_SYSTEMD = False
+
+try:
+    with open('/etc/mesosphere/setup-flags/repository-url') as f:
```

Comment:
setup-flags shouldn't be used here, it's only for `pkgpanda setup` / not guaranteed to be what we want in time. (In fact, I see it highly likely we want a different url when actually installing).

Practically we're going to want to move the packages into a in-cluster repository, then probably use the mesos-dns name to fetch them from there to individual hosts.